### PR TITLE
Update repair_wreck.sqf

### PR DESCRIPTION
**Pull request by heitor-sampaio**
- FIX: "Ship" vehicle class in wreck repair was missing (@heitor-sampaio).

**When merged this pull request will:**
- Changed line 27 to add  ships to repair wreck system.
- Added "Ships" vehicle class to wreck repair. Very important to Tanoa version.

**Final test:**
- [x] local
- [x] server

### CODE REVIEW

**Comment by Vdauphin** [comment]
Hello,
How do you recover a destroyed chip (mainly when it is destroyed in the middle of the ocean)?

I recommend to make it respawnable (http://vdauphin.github.io/HeartsAndMinds/Add-vehicle-or-boat-to-H&M-system#2-add-respawn-to-vehicle-or-boat)
Cheers

**Comment by heitor-sampaio** [comment]
Hello,

I can't recover vehicles from the middle of the ocean, but, i can from beachs. Sometimes players destroy their ships by hitting the docks or even hitting rocks on the beach.

We have a situation here, where players crashed their boats with other boats, then both boats turned into wrecks and they menaged to get the boats into the logistic point.

**Comment by Vdauphin** [comment]
If it is happening you can use submarine like here : 
https://github.com/Vdauphin/HeartsAndMinds/pull/310

**Comment by heitor-sampaio** [comment]
Other problem i face here is, when transporting vehicles with slingload and rope breaks in the middle of the ocean. Vehicles dont turn wrecks in this case. They just suffer a little damage. I trying to add eventhandler to addaction to recover this vehicles from the botton of the ocean by a diver.

**Comment by heitor-sampaio** [comment]
Yes, we use then. But, sometimes in shallow depths, its not possible to slingload other vehicles, because this system os intended to bem used on helicopters.

**Comment by Vdauphin** [comment]
>its not possible to slingload other vehicles, because this system os intended to bem used on helicopters.

It works with submarine as shown, what do you mean ?

**Comment by heitor-sampaio** [comment]
To hook when slingloading, the vehicles needs to be x meters above to the vehicle that will be slingloaded. In my case, x=7, so, If the depth is less than 7 meters, the hook action dont came up.

**Comment by Vdauphin** [comment]
then you can use an helicopter

**Comment by heitor-sampaio** [comment]
Sometimes yes, sometimes no. But recover vehicles isnt a big problem. The problem is that we can't repair boats wrecks on logistic point, wich ia solved here. As you suggested, i can add respawn to ships, but, here, we like harder difficulties.

**Comment by heitor-sampaio** [comment]
Sorry about my english errors. My phone keeps changing the words cause it os configured to portuguese. Also, my english isnt good at all.

**Comment by Vdauphin** [comment]
No problem, I understand!

**Comment by heitor-sampaio** [comment]
Thanks! Congratulations for jeep this mission updated, it's a very good one!

File: =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/log/repair_wreck.sqf [hunk 1]
```diff
@@ -24,7 +24,7 @@ params [
     ["_object", objNull, [objNull]]
 ];
 
-private _array = (nearestObjects [_object, ["LandVehicle", "Air"], 10]) select {!(
+private _array = (nearestObjects [_object, ["LandVehicle", "Air", "Ships"], 10]) select {!(
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/log/repair_wreck.sqf]:
```suggestion
private _array = (nearestObjects [_object, ["LandVehicle", "Air", "Ship"], 10]) select {!(
```

**Pull request merged**